src/main/java/thrift/logic/commands/ConvertCommand.java

```diff
@@ -18,11 +18,12 @@
 
     public static final String DEFAULT_VALUE = "1.00";
     public static final String DEFAULT_CURRENCY = "SGD";
+
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Converts amount "
             + "from the first specified currency to the rest of the specified currencies.\n"
             + "If no value is specified, it converts with a value of " + DEFAULT_VALUE + ".\n"
             + "If only one currency is specified, it converts from " + DEFAULT_CURRENCY + " to that currency.\n"
-            + "Parameters: [" + CliSyntax.PREFIX_VALUE + "amount] (must be a double) "
+            + "Parameters: [" + CliSyntax.PREFIX_VALUE + "AMOUNT] (must be a double) "
```

Comment:
`(up to 2 decimal places)` instead of `(must be a double)`

Need to change in `HELP_MESSAGE` as well